src/isomorphic/classic/element/ReactElement.js

```diff
@@ -162,6 +162,13 @@ var ReactElement = function(type, key, ref, self, source, owner, props) {
         writable: false,
         value: source,
       });
+      // _owner attribute would break shallow renderer equality checks.
+      Object.defineProperty(element, '_owner', {
+        configurable: false,
+        enumerable: false,
+        writable: false,
+        value: owner,
+      });
```

Comment:
Seb and I chatted about this idea in person. He didn't seem to, but I'll let him comment here (or I can follow up with him in person on Monday). Sure!